apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/exception/Exceptions.kt

```diff
@@ -99,8 +99,33 @@ class CacheMissException(val key: String, val fieldName: String? = null) : Apoll
 class HttpCacheMissException(message: String, cause: Exception? = null) : ApolloException(message = message, cause = cause)
 
 /**
- * Multiple exceptions happened, for an example with a [CacheFirst] fetch policy
- * [cause] might change in future where both exceptions will be added as suppressed exception
+ * Multiple exceptions happened. This happens typically when a [CacheFirst] request fails to fetch
+ * both the cache and the network. In that case, [ApolloCompositeException] is thrown, and you
+ * can access the underlying cache and network exceptions in [suppressedExceptions].
```

Comment:
Maybe also clarify that the underlying exceptions in `suppressedExceptions` always will be subclasses of `ApolloException` (see https://github.com/apollographql/apollo-kotlin/issues/4062#issuecomment-1110801107)

Reply:
comment and sample code updated.